lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -3179,9 +3193,10 @@ TypeSystemSwiftTypeRef::GetTypeBitAlign(opaque_compiler_type_t type,
   // alignment information out of DWARF. Because it is stored in the
   // Type object we need to look that up by name again.
   if (TypeSP type_sp = LookupTypeInModule(type))
-    return type_sp->GetLayoutCompilerType().GetTypeBitAlign(exe_scope);
+    if (type_sp->GetLayoutCompilerType().GetOpaqueQualType() != type)
```

Comment:
Suggestion: a comment saying when/why this inequality happens.

Reply:
It's again to avoid an infinite recursion. I'll add a comment.